### DISCUSSION TITLE
handle message at the same time

### DISCUSSION THREAD

**Discussion by bezzzz098** [socketio/socket.io#4785] [Q&A] [answered]
i wanna ask that are socket.io in general or flask -socket.io python (especially that i use) able to handle process at the same time?

for example in chat web app i have 4 client (a,b,c,d) hosted in 1 server which a is talking to b and c is talking to d. when in certain condition in order a to talk to b, it need 6 seconds on the other hand, c to d need only one second, if a talk to b and one second later c talk to d, do d need 6 second in order to recieve the message? ( 1+ 5 second from a to b) or only need 1 second cause a and b doesnt affect any other client?

**Comment by darrachequesne** [comment, marked as answer]
Hi! Messages between clients should not impact each other (provided that you don't reach the limit of your bandwidth, of course).

**Reply by ishaninfratech** [reply]
[hg

**Comment by ishaninfratech** [comment]
ty
